### DISCUSSION TITLE
Sybil Hunt

### DISCUSSION THREAD

**Discussion by Mithrandir0xYz** [zkSync-Community-Hub/zksync-developers#381] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

The application for community to report the suspicious sybil accounts to be reviewed manually...

### What is the specific issue or error you're encountering?

Once a report is submitted, itshould be rewiewed by a team or an automated system to verify the claim. The verification process should check for patterns that are indicative of Sybil behavior, such as multiple accounts from the same IP adress, similar transaction patterns or unnatural activity.

### Can you share the error messages or logs you're receiving, if any?

none

### Have you made any recent changes to the contract before encountering this issue?

none

### Are there any external libraries or contracts that your contract interacts with?

none

### Can you provide the relevant portions of your contract code where the issue is occurring?

none

### Have you tried to isolate the problem, and if so, what were the results?

none

### What steps have you already taken to try to resolve the issue?

none

### Additional Details

Can an application be made that can be reported by auxiliary Sybil applications to produce Sybil accounts and Sybil accounts that are safe for end users? Incentive rewards can also be given for reporting.

**Comment by npam90** [comment]
Yes, an application can be developed to allow the community to report suspicious Sybil accounts. This application can incorporate both manual review processes by a team and automated verification systems to detect patterns indicative of Sybil behavior. Incentive rewards for reporting can also be implemented to encourage community participation and enhance the effectiveness of the detection system.

**Comment by deosovitis** [comment]
The user is attempting to implement an application for the community to report suspicious Sybil accounts on the Mainnet. They are requesting assistance with implementing a verification process for these reports, which should check for patterns indicative of Sybil behavior. The user has not encountered any specific issues or errors, and has not made any recent changes to the contract. They are also considering offering incentive rewards for reporting. No repository link or additional details were provided.
